mintapi/api.py

```diff
@@ -537,23 +533,67 @@ def get_budgets(self):  # {{{
             headers=self.json_headers
         ).text)
 
-        # Make the skeleton return structure
-        budgets = {
-            'income': response['data']['income'][
-                str(max(map(int, response['data']['income'].keys())))
-            ]['bu'],
-            'spend': response['data']['spending'][
-                str(max(map(int, response['data']['income'].keys())))
-            ]['bu']
-        }
+        if hist is not None:  # version proofing api
+            def mos_to_yrmo(mos_frm_zero):
+                date_yr_mo = datetime(year=int(mos_frm_zero / 12),
+                                      month=mos_frm_zero % 12 + 1,
+                                      day=1).strftime("%Y%m")
+                return date_yr_mo
```

Comment:
Might as well just directly return the created datetime instead of creating a variable and then returning it on the next line (ie, return datetime(...))